packages/plugins/typescript/urql-graphcache/src/index.ts

```diff
@@ -253,7 +253,7 @@ export const plugin: PluginFunction<UrqlGraphCacheConfig, Types.ComplexPluginOut
   return {
     prepend: [imports],
     content: [
-      `export type WithTypename<T extends { __typename?: any }> = { [K in Exclude<keyof T, '__typename'>]?: T[K] } & { __typename: NonNullable<T['__typename']> };`,
+      `export type WithTypename<T extends { __typename?: any }> = T & { __typename: NonNullable<T['__typename']> };`,
```

Comment:
Instead of doing
```TypeScript
export type WithTypename<T extends { __typename?: any }> = Pick<T, Exclude<keyof T, '__typename'>> & { __typename: NonNullable<T['__typename']> };
```
I chose to replace `Pick<T, Exclude<keyof T, '__typename'>>` with `T`

Not sure if this change is okay, but let me know if you still want the change to be exactly as what you've requested!

Reply:
> I chose to replace `Pick<T, Exclude<keyof T, '__typename'>>` with `T`

yup, it seems to do the job!